src/components/VirtualThought.tsx

```diff
@@ -99,6 +100,7 @@ const VirtualThought = ({
   const fontSize = useSelector(state => state.fontSize)
   const note = useSelector(state => noteValue(state, thought.id))
   const ref = useRef<HTMLDivElement>(null)
+  const noOtherContexts = useSelector(state => isContextViewActive(state, simplePath))
```

Comment:
Make sure to update the function name to reflect the new functionality. I've updated it this time.